dio/lib/src/interceptor.dart

```diff
@@ -199,10 +199,10 @@ class Interceptor {
 
   /// Called when an exception was occurred during the request.
   void onError(
-    DioException err,
+    DioException error,
```

Comment:
Ah did I say it wrong? I mean we only change the type alias's parameter name (as you already did), but not changing the callback's.

Reply:
> This makes sense.

Oh, I didn't see that sentence just now and I resubmitted the changes. I thought you still wanted to change `err` to `error`, but only on the 6.0.0 branch. If it's okay to do it the way it was before, I can revert the previous commit.

Reply:
I redid the commit, hoping it's like this. Otherwise, I might have to speak in Chinese😁.